components/login/login_controller/login_controller.jsx

```diff
@@ -565,6 +565,7 @@ class LoginController extends React.PureComponent {
                         <div className={'form-group' + errorClass}>
                             <input
                                 id='loginId'
+                                data-testid='loginId'
```

Comment:
Weren't we asked to stop using test ids?

Reply:
Removed. It isn't a hard stop using them but rather a last resort. I thought I had no other option so I decided to use this but re-visiting it, I noticed I can use placeholder text instead.

This is the order to consider. I found out that I can use placeholder text.
![image](https://user-images.githubusercontent.com/17804942/102229849-37722a00-3eba-11eb-9ee3-0352d9a53fc3.png)